infra/app/app-config/env-config/identity-provider.tf

```diff
@@ -30,6 +33,10 @@ locals {
       verification_email_subject = null
     }
 
+    # Define variable names used across modules.
+    client_secret_ssm_name       = "/${var.app_name}-${var.environment}/identity-provider/client-secret"
+    user_pool_access_policy_name = "${var.app_name}-${var.environment}-cognito-access"
```

Comment:
Take a look at the comment I left on infra/modules/identity-provider/README.md, but I think we can have a new pattern that eliminates the need for config variables that are used primarily for dependency management rather than for actual configuration. tldr; I think we can move the definition of these things to something like a `modules/identity/interface/` module that can be used by both the module with data sources and the module that actually creates the resources